### PR TITLE
Get rid of temporary test file

### PR DESCRIPTION
**Pull request by belu**
When tests are run, a temporary file `testerFXML.fxml` is left behind.

This PR changes this by using a JUnit `TemporaryFolder` with `@Rule`.

### Issue
None.

### Progress

- [x] Change must not contain extraneous whitespace
- [x] License header year is updated, if required
- [x] Verify the contributor has signed [Gluon Individual Contributor License Agreement (CLA)](https://docs.google.com/forms/d/16aoFTmzs8lZTfiyrEm8YgMqMYaGQl0J8wA0VJE2LCCY)

### CODE REVIEW

File: kit/src/test/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMSaverUpdateImportInstructionsTest.java [hunk 1]
```diff
@@ -188,9 +192,9 @@ public void testCustomButton() {
         });
 
         assertTrue("fxml has import com.oracle.javafx.scenebuilder.kit.fxom.TestCustomButton",
-                imports.contains("com.oracle.javafx.scenebuilder.kit.fxom.TestCustomButton"));
+            imports.contains("com.oracle.javafx.scenebuilder.kit.fxom.TestCustomButton"));
```
Comment (abhinayagarwal) [inline kit/src/test/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMSaverUpdateImportInstructionsTest.java:195]: Please revert this change.

File: kit/src/test/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMSaverUpdateImportInstructionsTest.java [hunk 2]
```diff
@@ -188,9 +192,9 @@ public void testCustomButton() {
         });
 
         assertTrue("fxml has import com.oracle.javafx.scenebuilder.kit.fxom.TestCustomButton",
-                imports.contains("com.oracle.javafx.scenebuilder.kit.fxom.TestCustomButton"));
+            imports.contains("com.oracle.javafx.scenebuilder.kit.fxom.TestCustomButton"));
         assertFalse("fxml does not contain com.oracle.javafx.scenebuilder.*",
-                imports.contains("com.oracle.javafx.scenebuilder.*"));
+            imports.contains("com.oracle.javafx.scenebuilder.*"));
```
Comment (abhinayagarwal) [inline kit/src/test/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMSaverUpdateImportInstructionsTest.java:197]: Please revert this change.

File: kit/src/test/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMSaverUpdateImportInstructionsTest.java [hunk 3]
```diff
@@ -214,9 +218,9 @@ public void testImportsWithComments() {
         assertTrue("fifth glue node should be a comment", fxomDocument.getGlue().getHeader().get(4) instanceof GlueCharacters);
 
         assertTrue("fxml does not contain javafx.scene.control.ComboBox",
-                imports.contains("javafx.scene.control.ComboBox"));
+            imports.contains("javafx.scene.control.ComboBox"));
```
Comment (abhinayagarwal) [inline kit/src/test/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMSaverUpdateImportInstructionsTest.java:221]: Please revert this change.

File: kit/src/test/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMSaverUpdateImportInstructionsTest.java [hunk 4]
```diff
@@ -214,9 +218,9 @@ public void testImportsWithComments() {
         assertTrue("fifth glue node should be a comment", fxomDocument.getGlue().getHeader().get(4) instanceof GlueCharacters);
 
         assertTrue("fxml does not contain javafx.scene.control.ComboBox",
-                imports.contains("javafx.scene.control.ComboBox"));
+            imports.contains("javafx.scene.control.ComboBox"));
         assertTrue("fxml does not contain javafx.scene.layout.AnchorPane",
-                imports.contains("javafx.scene.layout.AnchorPane"));
+            imports.contains("javafx.scene.layout.AnchorPane"));
```
Comment (abhinayagarwal) [inline kit/src/test/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMSaverUpdateImportInstructionsTest.java:223]: Please revert this change.

**Comment by abhinayagarwal** [comment]
I have left a few comments about reverting the changes which has nothing to do with this PR. Other than that the PR lgtm.

**Comment by belu** [comment]
Hi @abhinayagarwal, reverted the style changes.

**Review by abhinayagarwal — APPROVED** [review]
